packages/image_picker/image_picker_ios/example/ios/RunnerUITests/ImagePickerFromGalleryUITests.m

```diff
@@ -59,103 +59,95 @@ - (void)tearDown {
   [self.app terminate];
 }
 
-- (void)testPickingFromGallery {
-  [self launchPickerAndPick];
-}
-
 - (void)testCancel {
-  [self launchPickerAndCancel];
-}
-
-- (void)launchPickerAndCancel {
   // Find and tap on the pick from gallery button.
-  NSPredicate *predicateToFindImageFromGalleryButton =
-      [NSPredicate predicateWithFormat:@"label == %@", @"image_picker_example_from_gallery"];
-
   XCUIElement *imageFromGalleryButton =
-      [self.app.otherElements elementMatchingPredicate:predicateToFindImageFromGalleryButton];
+      self.app.otherElements[@"image_picker_example_from_gallery"].firstMatch;
   if (![imageFromGalleryButton waitForExistenceWithTimeout:kElementWaitingTime]) {
     os_log_error(OS_LOG_DEFAULT, "%@", self.app.debugDescription);
     XCTFail(@"Failed due to not able to find image from gallery button with %@ seconds",
             @(kElementWaitingTime));
   }
 
-  XCTAssertTrue(imageFromGalleryButton.exists);
```

Comment:
Don't need to check that it exists, `waitForExistenceWithTimeout` right above does this already.